SingularConfig.js

```diff
@@ -0,0 +1,28 @@
+export class SingularConfig {
+    apikey;
+    secret;
+    sessionTimeout;
+    customUserId;
+    singularLinkHandler;
+
+    constructor(apikey, secret) {
+        this.apikey = apikey;
+        this.secret = secret;
+        this.sessionTimeout = -1; // default -1, uses default timeout (60s)
```

Comment:
why not just use `null`? better than magic numbers

Reply:
I can't pass null because the converted bridged types don't support null